.dockerignore

```diff
@@ -0,0 +1,8 @@
+.git
+.github
+.dockerignore
+.gitignore
+Dockerfile
+example.yaml
+LICENSE
```

Comment:
The license says:

```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```

Saying "MIT" isn't enough. So you can't ignore it. It should usually be copied in the root of the image to correctly comply with it. (This is mostly why I switched some of my more recent code to 0BSD, as this is a pain to do correctly.)

Reply:
I have now copied it into the Docker image.